### PR TITLE
Add option for disabling arrow icon in collapse panel

### PR DESCRIPTION
**Pull request by apieceofbart**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you propose PR to right branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [x] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [x] Update API docs for the component.
  * [x] Update/Add demo to demonstrate new feature.
  * [x] Update TypeScript definition for the component.
  * [x] Add unit tests for the feature.

This is in regards to my proposal described here:
#8526

Obviously this requires checking and Chinese translation but I think it's a good starting point.

### CODE REVIEW

File: components/collapse/CollapsePanel.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  disabled?: boolean;
+  className?: string;
+  style?: React.CSSProperties;
+  showArrow?: boolean;
+  prefixCls?: string;
+}
+
+export default class CollapsePanel extends React.Component<CollapsePanelProps, {}> {
+  render() {
+    const { prefixCls, className = '', showArrow = true } = this.props;
```
Comment (apieceofbart) [inline components/collapse/CollapsePanel.tsx]: not sure if all of this is needed

File: components/collapse/style/index.less [hunk 2]
```diff
@@ -54,6 +54,12 @@
         }
       }
     }
+
+    &.@{collapse-prefix-cls}-no-arrow {
+      > .@{collapse-prefix-cls}-header {
+        padding-left: 12px;
```
Comment (apieceofbart) [inline components/collapse/style/index.less]: this is to move the text left so there's no blank space when icon is missing

File: components/collapse/demo/noarrow.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+order: 4
+title:
+  zh-CN: TODO
```
Comment (apieceofbart) [inline components/collapse/demo/noarrow.md]: missing translation

File: components/collapse/demo/noarrow.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+  en-US: No arrow
+---
+
+## zh-CN
+
+TODO
```
Comment (apieceofbart) [inline components/collapse/demo/noarrow.md]: missing translation

**Comment by apieceofbart** [comment]
tests fail for `select` component with react 16 - I don't think it's related to my changes

**Comment by afc163** [comment]
You can try rebase `feature-3.1`.

**Comment by apieceofbart** [comment]
hmm I think I'm up to date with feature-3.1..

**Comment by afc163** [comment]
Try it again.

**Comment by apieceofbart** [comment]
done.

**Comment by afc163** [comment]
image

**Comment by apieceofbart** [comment]
@afc163 was that an issue? I added my work email

**Pull request merged**